lib/tether/ops_linux.go

```diff
@@ -57,6 +57,7 @@ const (
 	pciDevPath         = "/sys/bus/pci/devices"
 	nfsFileSystemType  = "nfs"
 	ext4FileSystemType = "ext4"
+	bridgeTableNumber  = 201
```

Comment:
Is this truly a constant? or should is there a way to look up what the table number is? I'm just concerned that down the road linux renumbers these things by default and we should think about defensive code for that.